### ISSUE TITLE
Release version with bundled Tailwind CSS v3.4

### ISSUE THREAD

**Issue by rhcarvalho** [tailwindlabs/tailwindcss-intellisense#891]
**What version of VS Code are you using?**

v1.85.1

**What version of Tailwind CSS IntelliSense are you using?**

Latest release v0.10.2 and pre-release v0.11.32

**What version of Tailwind CSS are you using?**

For example: v3.4.0

**What package manager are you using?**

Using Tailwind in a Phoenix project, which relies on the `tailwindcss` CLI instead of JavaScript tooling.

**What operating system are you using?**

Linux

**Tailwind config**

N/A

**VS Code settings**

N/A

**Reproduction URL**

N/A

**Describe your issue**

This is the output from the extension logs:

```
[Global] Creating projects: [{"folder":"...","configPath":".../assets/tailwind.config.js","isUserConfigured":false,"documentSelector":[{"pattern":".../assets/tailwind.config.js","priority":0},{"pattern":".../assets/**","priority":3},{"pattern":".../**","priority":4}]}]
[Global] Adding watch patterns: .../assets/tailwind.config.js, .../assets, ...
[assets/tailwind.config.js] Initializing...
[assets/tailwind.config.js] Failed to load workspace modules.
[assets/tailwind.config.js] Using bundled version of `tailwindcss`: v3.3.0
[assets/tailwind.config.js] Using bundled version of `postcss`: v8.4.31
[assets/tailwind.config.js] Building...
```

Not surprisingly it's using version v3.3.0, so after upgrading to v3.4.0 I don't have intellisense completions for the new features `size-X`, etc.

I see the bundled version was bumped in #889, so would it be possible to release a new version to ship that?

Thanks!

**Comment by thecrypticace** [comment]
It's on my list to do today! (probably closer to this evening)

**Comment by rhcarvalho** [comment]
Thanks, Jordan! Happy holidays!

**Comment by thecrypticace** [comment]
Released as v0.10.4! Happy holidays!

<img width="1143" alt="Screenshot 2023-12-21 at 17 06 16" src="https://github.com/tailwindlabs/tailwindcss-intellisense/assets/614993/5c7f2524-46e2-4a1d-bab4-a93d43975aa9">
